### PR TITLE
Flav/sparkle dropdown

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR extracts the `NewDropdown` component from https://github.com/dust-tt/dust/pull/7475.

This new component is not yet used but is meant to replace all existing `Dropdown`.

Example:

image

### CODE REVIEW

File: sparkle/tailwind.config.js [hunk 1]
```diff
@@ -232,11 +287,7 @@ module.exports = {
       backgroundColor: ["dark"],
     },
   },
-  plugins: [
-    require("@tailwindcss/forms"),
-    require("tailwind-scrollbar-hide"),
-    require("tailwindcss-animate"),
```
Comment (JulesBelveze) [inline sparkle/tailwind.config.js:238]: Can you add back `tailwindcss-animate`?

File: sparkle/src/components/NewDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    ref={ref}
+    className={cn(
+      menuStyleClasses.item,
+      inset ? menuStyleClasses.inset : "",
+      className || ""
+    )}
+    {...props}
+  >
+    {label && (
+      <div className="s-flex s-flex-row s-gap-1">
```
Comment (JulesBelveze) [inline sparkle/src/components/NewDropdown.tsx:137]: Could we add `s-items-center`?

File: sparkle/src/components/NewDropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    )}
+    {...props}
+  >
+    <span className={menuStyleClasses.subTrigger.span}>
+      <DropdownMenuPrimitive.ItemIndicator>
+        <Icon size="xs" visual={CircleIcon} />
+      </DropdownMenuPrimitive.ItemIndicator>
+    </span>
+    {label && (
+      <div className="s-flex s-flex-row s-gap-1">
```
Comment (JulesBelveze) [inline sparkle/src/components/NewDropdown.tsx:197]: Same here, could we add `s-items-center`?

**Review by JulesBelveze — COMMENTED** [review]
Left a couple of comments, looks 🔥 

Note: let me merge the `Popover` PR and move the misused `Dropdown` to use it first, so that we avoid going over the same components twice 😇

File: sparkle/tailwind.config.js (hunk 1 above)
Comment (flvndvd) [reply sparkle/tailwind.config.js]: Ouspie 🙈

File: sparkle/src/components/NewDropdown.tsx (hunk 2 above)
Comment (flvndvd) [reply sparkle/src/components/NewDropdown.tsx:137]: Seen with Ed, we want to keep the icon aligned with the title.

Comment (flvndvd) [reply sparkle/src/components/NewDropdown.tsx]: Updated, good catch!

**Review by JulesBelveze — APPROVED** [review]
Looking good 💯

**Pull request merged**